text/0149-rfc-1-renewal-adjustment.md

```diff
@@ -0,0 +1,168 @@
+
+The Dutch auction of a sale, has an end price, a target price in the middle and
+a starting price. The target price is ten times the end price and the start
+price is ten times the target price (or 100 times the end price).
+
+The target price is the expected market price of a core. To be more precise,
+what happens is that, for a sale X+1 the end price is set to the clearing price
+of sale X divided by 10. This results in a maximum adjustment of the price
+curve (and the end price) by a factor of 10 each sale.
+
+To illustrate this with actual numbers: Let's consider an end price of 10 DOT,
+then if someone bought all the available cores at 1000 DOT (starting price),
+then the end price in the next sale would become 100 DOT.
+
+Prior to this RFC, all of this hardly mattered for existing tenants, which
+keeps being the case, except for extreme market changes - or if you happened to
+be lucky to buy below the target price. In particular, in the above example,
+all cores were sold at 1000 DOT, then tenants who were lucky holding a core at
+10 DOT, would now get bumped to 100 DOT. Tenants who bought at the current
+target price or above would still not be affected.
+
+In other words, we limit price predictability: If the market experiences a bump
+of 100 times the current price, renewals will be affected by a factor of 10.
+Even in that extreme situation, renewal prices would still be 10x more stable than
+the market: Therefore it is fair to say that this RFC is maintaining
+predictability within reasonable bounds.
+
+So, while predictability is mostly maintained, this RFC ensures that renewals
+don't get detached too much from the current market, with the following
+effects:
+
+1. Attacking the system is becoming very expensive. Let's assume 10 cores are
+   for sale at an end price of 10 DOT, then an attacker would need to invest
+10_000 DOT to drive up renewal prices to 100 DOT. Not only is this expensive in
+itself with limited effect, the attacker would also have driven up the price
+for his own renewals: Core hoarding for price manipulation is no longer
+profitable. An attacker would be at least as much affected as the honest
+parties he is targeting. In practice likely more, as core hoarding hardly means
+holding only one or two cores.
+2. In the honest scenario, where we are just experiencing great adoption: Once
+   cores become scarce, prices would go up and renewals would be affected too.
+They are still favored over new entrants (they pay 10x less), but as prices
+keep increasing, eventually the least profitable projects will give up their
+cores, freeing up resources for new entrants. 
+
+
+Recommendations for deployment:
+
+1. Deploy with a price curve enforcing a minimum end price, we suggest 10 DOT
+   and 1 KSM. This is to ensure prompt recovery to sound market conditions.
+2. Together with a minimum price, it should be safe to add more cores. We
+   recommend adding at least 10 additional cores, 20 would be better. This
```

Comment:
```suggestion
   recommend adding at least 10 additional cores, 20 would be better. This
```

Maybe it is best to specify network-specific numbers for the number of cores to add, since we have more headroom on Polkadot than on Kusama, and with more cores we avoid the low-liquidity risks of the model more effectively (while still having a small reserve)